src/main/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrDayPageTask.java

```diff
@@ -0,0 +1,77 @@
+package de.mediathekview.mserver.crawler.wdr.tasks;
+
+import de.mediathekview.mserver.base.Consts;
+import de.mediathekview.mserver.base.utils.UrlUtils;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import de.mediathekview.mserver.crawler.basic.TopicUrlDTO;
+import de.mediathekview.mserver.crawler.wdr.WdrConstants;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class WdrDayPageTask extends AbstractDocumentTask<TopicUrlDTO, CrawlerUrlDTO> {
+
+  private static final String ENTRY_SELECTOR =
+      "div.section > div.con > div.mod > div.boxCon > div.box";
+  private static final String URL_SELECTOR = "div.hideTeasertext > a";
+  private static final String TOPIC_SELECTOR = "h3.ressort > a";
+
+  public WdrDayPageTask(
+      AbstractCrawler aCrawler, ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDtos) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)